i18n/locale-info.php

```diff
@@ -3,326 +3,3980 @@
+		'currency_code'  => 'WST',
 		'currency_pos'   => 'left',
 		'thousand_sep'   => ',',
 		'decimal_sep'    => '.',
+		'num_decimals'   => 2,
+		'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
+		'direction'      => 'ltr',
+		'default_locale' => 'en_WS',
+		'name'           => 'Samoan tālā',
+		'singular'       => 'Samoan tala',
+		'plural'         => 'Samoan tala',
+		'short_symbol'   => null,
+		'locales'        => $locales['WST'],
+	),
+	'XK' => array(
+		'currency_code'  => 'EUR',
+		'currency_pos'   => 'right_space',
+		'thousand_sep'   => ' ',
+		'decimal_sep'    => ',',
+		'num_decimals'   => 2,
+		'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
+		'direction'      => 'ltr',
+		'default_locale' => 'sq_XK',
+		'name'           => 'Euro',
+		'singular'       => 'euro',
+		'plural'         => 'euros',
+		'short_symbol'   => '€',
+		'locales'        => $locales['EUR'],
+	),
+	'YE' => array(
+		'currency_code'  => 'YER',
+		'currency_pos'   => 'right_space',
+		'thousand_sep'   => '.',
+		'decimal_sep'    => ',',
 		'num_decimals'   => 0,
 		'weight_unit'    => 'kg',
 		'dimension_unit' => 'cm',
+		'direction'      => 'rtl',
+		'default_locale' => 'ar_YE',
+		'name'           => 'Yemeni rial',
+		'singular'       => 'Yemeni rial',
+		'plural'         => 'Yemeni rials',
+		'short_symbol'   => null,
+		'locales'        => $locales['YER'],
 	),
-	'US' => array(
-		'currency_code'  => 'USD',
+	'YT' => array(
+		'currency_code'  => 'EUR',
+		'currency_pos'   => 'right_space',
+		'thousand_sep'   => ' ',
+		'decimal_sep'    => ',',
+		'num_decimals'   => 2,
+		'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
+		'direction'      => 'ltr',
+		'default_locale' => 'fr_YT',
+		'name'           => 'Euro',
+		'singular'       => 'euro',
+		'plural'         => 'euros',
+		'short_symbol'   => '€',
+		'locales'        => $locales['EUR'],
+	),
+	'ZA' => array(
+		'currency_code'  => 'ZAR',
+		'currency_pos'   => 'left',
+		'thousand_sep'   => ' ',
+		'decimal_sep'    => ',',
+		'num_decimals'   => 2,
+		'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
+		'direction'      => 'ltr',
+		'default_locale' => 'en_ZA',
+		'name'           => 'South African rand',
+		'singular'       => 'South African rand',
+		'plural'         => 'South African rand',
+		'short_symbol'   => 'R',
+		'locales'        => $locales['ZAR'],
+	),
+	'ZM' => array(
+		'currency_code'  => 'ZMW',
 		'currency_pos'   => 'left',
 		'thousand_sep'   => ',',
 		'decimal_sep'    => '.',
 		'num_decimals'   => 2,
-		'weight_unit'    => 'oz',
-		'dimension_unit' => 'in',
+		'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
+		'direction'      => 'ltr',
+		'default_locale' => 'en_ZM',
+		'name'           => 'Zambian kwacha',
+		'singular'       => 'Zambian kwacha',
+		'plural'         => 'Zambian kwachas',
+		'short_symbol'   => 'ZK',
+		'locales'        => $locales['ZMW'],
 	),
-	'ZA' => array(
-		'currency_code'  => 'ZAR',
+	'ZW' => array(
+		'currency_code'  => 'USD',
```

Comment:
Is this normal to have non US countries to use USD? I see this in other areas as well.

Reply:
Yes, that surprised me too, but it seems some countries use directly USD. I've checked it from this list of countries: https://www.localeplanet.com/icu/iso3166.html and it seems that way.

Reply:
Hmmm I wonder if that information is correct.  If you look it up, they use Zimbabwean dollars. While they may allow USD currency, their official country currency is Zimbabwean dollars. This is quite confusing... :(

Reply:
This is the currency history of Zimbabwe according to the CLDR repository (trunk) https://github.com/unicode-org/cldr-json/blob/39205fdc027c935e52a268afcf01cd240f3c442d/cldr-json/cldr-core/supplemental/currencyData.json#L3468-L3498, the values with a `_to` key shows deprecated currencies, without a `_to` key shows the current ones.

Reply:
Yeah I saw that hence it is confusing as other documents state something else. However in the interest of this, we're following CLDR so we can only go by what they state. So I am ok with this change.